### ISSUE TITLE
[Investigation] Is the ENA scraper broken?

### ISSUE THREAD

**Issue by Miserlou** [AlexsLemonade/refinebio#1174]
I think the ENA scraper might be broken. https://circleci.com/gh/AlexsLemonade/refinebio/15332

```
WARNING:data_refinery_foreman.surveyor.harmony:Cannot determine sample title!
2019-03-25 18:06:57,551 local [volume: 0] data_refinery_foreman.surveyor.harmony WARNING [sample: {'<!DOCTYPE html><!--[if lt IE 7]><html class="no-js ie6 oldie" lang="en"><![endif]--><!--[if IE 7]><html class="no-js ie7 oldie" lang="en"><![endif]--><!--[if IE 8]><html class="no-js ie8 oldie" lang="en"><![endif]--><!--[if gt IE 8]><!--><html class="no-js" lang="en"><!--<![endif]--><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><title>Sorry, this page is not available &lt; ArrayExpress &lt; EMBL-EBI</title><meta name="description" content="EMBL-EBI"><meta name="keywords" content="bioinformatics, europe, institute"><meta name="author" content="EMBL-EBI"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="stylesheet" href="//www.ebi.ac.uk/web_guidelines/EBI-Framework/v1.1/libraries/foundation-6/css/foundation.css" type="text/css"><link rel="stylesheet" href="//www.ebi.ac.uk/web_guidelines/EBI-Framework/v1.1/css/ebi-global.css" type="text/css"><link rel="stylesheet" href="//www.ebi.ac.uk/web_guidelines/EBI-Icon-fonts/v1.1/fonts.css" type="text/css"><link rel="stylesheet" href="//www.ebi.ac.uk/web_guidelines/EBI-Framework/v1.1/css/theme-ebi-services-about.css" type="text/css"><link rel="stylesheet" href="/arrayexpress/assets/stylesheets/font-awesome.css" type="text/css"><link rel="stylesheet" href="/arrayexpress/assets/stylesheets/ae-common-1.0.161024.css" type="text/css"><script src="//www.ebi.ac.uk/web_guidelines/js/libs/modernizr.custom.49274.js"></script></head><body class="level2"><div id="skip-to"><ul><li><a href="#content" title="">Skip to main content</a></li><li><a href="#local-nav" title="">Skip to local navigation</a></li><li><a href="#global-nav" title="">Skip to EBI global navigation menu</a></li><li><a href="#global-nav-expanded" title="" class="row"></a></li></ul></div><div id="wrapper"><header><div><div id="local-masthead" class="meta-background-image" data-sticky="true" data-sticky-on="large" data-top-anchor="180" data-btm-anchor="300000"><div id="global-masthead" class="clearfix"><a href="//www.ebi.ac.uk/" title="Go to the EMBL-EBI homepage"><span class="ebi-logo"></span></a><nav><div class="row"><ul id="global-nav" class="menu"><li id="home-mobile" class=""><a href="//www.ebi.ac.uk"></a></li><li id="home"><a href="//www.ebi.ac.uk"><i class="icon icon-generic" data-icon="H"></i> EMBL-EBI</a></li><li id="services" class="active"><a href="//www.ebi.ac.uk/services"><i class="icon icon-generic" data-icon="("></i> Services</a></li><li id="research"><a href="//www.ebi.ac.uk/research"><i class="icon icon-generic" data-icon=")"></i> Research</a></li><li id="training"><a href="//www.ebi.ac.uk/training"><i class="icon icon-generic" data-icon="t"></i> Training</a></li><li id="about"><a href="//www.ebi.ac.uk/about"><i class="icon icon-generic" data-icon="i"></i> About us</a></li><li id="search"><a href="#" data-toggle="search-global-dropdown"><i class="icon icon-functional" data-icon="1"></i><span class="show-for-small-only">Search</span></a><div id="search-global-dropdown" class="dropdown-pane" data-dropdown="" data-options="closeOnClick:true;"><form id="global-search" name="global-search" action="/ebisearch/search.ebi" method="GET"><fieldset><div class="input-group"><input type="text" name="query" id="global-searchbox" class="input-group-field" placeholder="Search all of EMBL-EBI"><div class="input-group-button"><input type="submit" name="submit" value="Search" class="button"><input type="hidden" name="db" value="allebi" checked><input type="hidden" name="requestFrom" value="global-masthead" checked></div></div></fieldset></form></div></li><li class="float-right show-for-medium embl-selector"><button class="button" type="button" data-toggle="embl-dropdown">Hinxton</button><div id="embl-dropdown" class="dropdown-pane" data-dropdown="" data-options="closeOnClick:true;">': "  _gaq.push(['_setAccount', 'UA-21742948-1']);"}]: Cannot determine sample title!
```

That's almost certainly not supposed to be the JSON we're extracting a title from.

**Comment by Miserlou** [comment]
```
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 982, in emit
    stream.write(msg)
UnicodeEncodeError: 'ascii' codec can't encode character '\u2019' in position 4416: ordinal not in range(128)
Call stack:
  File "/usr/local/bin/coverage", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/coverage/cmdline.py", line 753, in main
    status = CoverageScript().command_line(argv)
  File "/usr/local/lib/python3.5/dist-packages/coverage/cmdline.py", line 488, in command_line
    return self.do_run(options, args)
  File "/usr/local/lib/python3.5/dist-packages/coverage/cmdline.py", line 627, in do_run
    self.run_python_file(filename, args)
  File "/usr/local/lib/python3.5/dist-packages/coverage/execfile.py", line 184, in run_python_file
    exec(code, main_mod.__dict__)
  File "manage.py", line 23, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/commands/test.py", line 26, in run_from_argv
    super().run_from_argv(argv)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/base.py", line 316, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/base.py", line 353, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/commands/test.py", line 56, in handle
    failures = test_runner.run_tests(test_labels)
  File "/usr/local/lib/python3.5/dist-packages/django/test/runner.py", line 606, in run_tests
    result = self.run_suite(suite)
  File "/usr/local/lib/python3.5/dist-packages/django/test/runner.py", line 572, in run_suite
    return runner.run(suite)
  File "/usr/lib/python3.5/unittest/runner.py", line 176, in run
    test(result)
  File "/usr/lib/python3.5/unittest/suite.py", line 84, in __call__
    return self.run(*args, **kwds)
  File "/usr/lib/python3.5/unittest/suite.py", line 122, in run
    test(result)
  File "/usr/local/lib/python3.5/dist-packages/django/test/testcases.py", line 206, in __call__
    super().__call__(result)
  File "/usr/lib/python3.5/unittest/case.py", line 648, in __call__
    return self.run(*args, **kwds)
  File "/usr/lib/python3.5/unittest/case.py", line 600, in run
    testMethod()
  File "/home/user/data_refinery_foreman/surveyor/test_harmony.py", line 211, in test_sdrf_big
    harmonized = harmonize(metadata)
  File "/home/user/data_refinery_foreman/surveyor/harmony.py", line 312, in harmonize
    logger.warn("Cannot determine sample title!", sample=sample)
Message: 'Cannot determine sample title!'
Arguments: ()
WARNING:data_refinery_foreman.surveyor.harmony:Cannot determine sample title!
```

**Comment by kurtwheeler** [comment]
I'm looking into this. I think it's Array Express, not ENA though.

**Comment by Miserlou** [comment]
Yes, I should have said EMBL-ENI. If only there were a handy graphic and guide for me to reference.

**Comment by kurtwheeler** [comment]
Closed by #1177
